configs/patch.json

```diff
@@ -0,0 +1,19 @@
+{
+  "roles": [
+    {
+      "name": "Patchman Access",
+      "description": "An all access role which grants read and write permissions.",
+      "system": true,
+      "platform_default": true,
+      "version": 2,
+      "access": [
+        {
+          "permission": "patch:*:*"
+        },
+        {
+          "permission": "inventory:*:*"
```

Comment:
Hey @semtexzv I just realized, we have [a ticket to remove all of the implicit inventory permissions.](https://projects.engineering.redhat.com/browse/RHCLOUD-4775) 

Could you pull that permission out? Then we should be good. 😄